lib/hound/internal_helpers.ex

```diff
@@ -79,15 +79,7 @@ defmodule Hound.InternalHelpers do
       ]
     ]
 
-    if driver_info[:driver] == "phantomjs" do
-      if Enum.member?(unsupported_features[:phantomjs], feature) do
-        false
-      else
-        true
-      end
-    else
-      true
-    end
+    driver_info[:driver] == "phantomjs" && !Enum.member?(unsupported_features[:phantomjs], feature)
```

Comment:
We could make it a module attribute but IMO we should leave it there since its only used in once place